Gemfile

```diff
@@ -14,6 +14,7 @@ end
 
 # override .gemspec deps - may issue warning depending on Bundler version
 gem "facter", *location_for(ENV['FACTER_LOCATION']) if ENV.has_key?('FACTER_LOCATION')
+gem "facter-ng"
```

Comment:
Could you add the `@location_for` logic that way we can point it to a local checkout of facter-ng?